client/README.md

```diff
@@ -36,10 +38,41 @@ npm run build
 npm run start
 ```
 
-Open your browser to `localhost:8000`.
+This will show you the default version of the app, where you can "buy" certain
+available products.
+
+Open your browser to `localhost:8081` to explore more.
+
+### Client variants
+
```

Comment:
The default variant of Avocano is an example implementation of a dynamic webapp that when you click on Buy, explains that this is not a real product website.